<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.ebay.automation.pagetests.LoginPageTest" tests="8" time="107.910" ignored="5" failures="4" timestamp="23 Jul 2020 12:45:15 GMT" skipped="4" hostname="LAPTOP-JOPOHLMU" errors="0">
  <testcase name="initialSetup" time="66.255" classname="com.ebay.automation.pagetests.LoginPageTest">
    <failure type="java.lang.AssertionError" message="Exception from TestBase.initialSetup">
      <![CDATA[java.lang.AssertionError: Exception from TestBase.initialSetup
	at org.testng.Assert.fail(Assert.java:83)
	at com.ebay.automation.base.TestBase.initialSetup(TestBase.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:590)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.Exception: Exception from WebDriverFactory.CreateDriver: [Ljava.lang.StackTraceElement;@1c4e3c2
	at com.ebay.automation.utils.WebDriverFactory.createDriver(WebDriverFactory.java:88)
	at com.ebay.automation.base.TestBase.initialSetup(TestBase.java:28)
	... 22 more
]]>
    </failure>
  </testcase> <!-- initialSetup -->
  <testcase name="initialSetup" time="0.000" classname="com.ebay.automation.pagetests.LoginPageTest">
    <failure type="java.lang.AssertionError" message="Exception from TestBase.initialSetup">
      <![CDATA[java.lang.AssertionError: Exception from TestBase.initialSetup
	at org.testng.Assert.fail(Assert.java:83)
	at com.ebay.automation.base.TestBase.initialSetup(TestBase.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:590)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.Exception: Exception from WebDriverFactory.CreateDriver: [Ljava.lang.StackTraceElement;@7cf47cb4
	at com.ebay.automation.utils.WebDriverFactory.createDriver(WebDriverFactory.java:88)
	at com.ebay.automation.base.TestBase.initialSetup(TestBase.java:28)
	... 22 more
]]>
    </failure>
  </testcase> <!-- initialSetup -->
  <testcase name="afterClass" time="40.912" classname="com.ebay.automation.pagetests.LoginPageTest">
    <failure type="java.lang.AssertionError" message="Exception from TestBase.afterClass">
      <![CDATA[java.lang.AssertionError: Exception from TestBase.afterClass
	at org.testng.Assert.fail(Assert.java:83)
	at com.ebay.automation.base.TestBase.afterClass(TestBase.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:712)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at com.ebay.automation.base.TestBase.afterClass(TestBase.java:42)
	... 22 more
]]>
    </failure>
  </testcase> <!-- afterClass -->
  <testcase name="assertLoginPageTitle" time="0.370" classname="com.ebay.automation.pagetests.LoginPageTest">
    <skipped/>
  </testcase> <!-- assertLoginPageTitle -->
  <testcase name="searchForProduct" time="0.002" classname="com.ebay.automation.pagetests.LoginPageTest">
    <skipped/>
  </testcase> <!-- searchForProduct -->
  <testcase name="login" time="0.368" classname="com.ebay.automation.pagetests.LoginPageTest">
    <skipped/>
  </testcase> <!-- login -->
  <testcase name="signUp" time="0.003" classname="com.ebay.automation.pagetests.LoginPageTest">
    <skipped/>
  </testcase> <!-- signUp -->
  <testcase name="afterClass" time="0.000" classname="com.ebay.automation.pagetests.LoginPageTest">
    <failure type="java.lang.AssertionError" message="Exception from TestBase.afterClass">
      <![CDATA[java.lang.AssertionError: Exception from TestBase.afterClass
	at org.testng.Assert.fail(Assert.java:83)
	at com.ebay.automation.base.TestBase.afterClass(TestBase.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:712)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at com.ebay.automation.base.TestBase.afterClass(TestBase.java:42)
	... 22 more
]]>
    </failure>
  </testcase> <!-- afterClass -->
  <testcase name="login" classname="com.ebay.automation.pagetests.LoginPageTest">
    <skipped/>
  </testcase> <!-- login -->
  <testcase name="assertLoginPageTitle" classname="com.ebay.automation.pagetests.LoginPageTest">
    <skipped/>
  </testcase> <!-- assertLoginPageTitle -->
  <testcase name="searchForProduct" classname="com.ebay.automation.pagetests.LoginPageTest">
    <skipped/>
  </testcase> <!-- searchForProduct -->
  <testcase name="signUp" classname="com.ebay.automation.pagetests.LoginPageTest">
    <skipped/>
  </testcase> <!-- signUp -->
  <testcase name="openPurchaseHistory" classname="com.ebay.automation.pagetests.HomePageTest">
    <skipped/>
  </testcase> <!-- openPurchaseHistory -->
</testsuite> <!-- com.ebay.automation.pagetests.LoginPageTest -->
